client/web/src/search/results/SearchResultsInfoBar.tsx

```diff
@@ -59,6 +61,9 @@ export interface SearchResultsInfoBarProps
     setSidebarCollapsed: (collapsed: boolean) => void
 
     isSourcegraphDotCom: boolean
+
+    isToggleEnabled: boolean
+    onToggleRanking: (newValue: boolean) => void
```

Comment:
nit suggestions for naming to match React naming conventions (eg. from `useState`)

```suggestion
    isRankingEnabled: boolean
    setRankingEnabled: (enabled: boolean) => void
```

Reply:
Done.